cdap-common/bin/common.sh

```diff
@@ -267,6 +267,10 @@ cdap_set_hive_classpath() {
     # classpath from it and export it to use it in the launch command
     if [ -n "${HIVE_HOME}" -a -n "${HIVE_CONF_DIR}" -a -n "${HADOOP_CONF_DIR}" ]; then
       EXPLORE_CONF_FILES=$(ls -1dF ${HIVE_CONF_DIR}/* ${HADOOP_CONF_DIR}/* | sed -e '/\/$/d' | tr '\n' ':')
+      if [ -n "${TEZ_HOME}" ]; then
```

Comment:
This will work and will simply ignore the Tez stuff if it isn't present. It will still require the end user on every HDP cluster to manually configure these in `cdap-env.sh` before starting CDAP.

```diff
diff --git a/cdap-common/bin/common.sh b/cdap-common/bin/common.sh
index f21a90d..d72b5f6 100755
--- a/cdap-common/bin/common.sh
+++ b/cdap-common/bin/common.sh
@@ -266,8 +266,8 @@ cdap_set_hive_classpath() {
     # If Hive classpath is successfully determined, derive explore
     # classpath from it and export it to use it in the launch command
     if [ -n "${HIVE_HOME}" -a -n "${HIVE_CONF_DIR}" -a -n "${HADOOP_CONF_DIR}" ]; then
-      EXPLORE_CONF_FILES=$(ls -1dF ${HIVE_CONF_DIR}/* ${HADOOP_CONF_DIR}/* | sed -e '/\/$/d' | tr '\n' ':')
-      EXPLORE_CLASSPATH=$(ls -1 ${HIVE_HOME}/lib/hive-exec-* ${HIVE_HOME}/lib/*.jar | tr '\n' ':')
+      EXPLORE_CONF_FILES=$(ls -1dF ${HIVE_CONF_DIR}/* ${HADOOP_CONF_DIR}/* ${TEZ_CONF_DIR}/* 2>/dev/null | sed -e '/\/$/d' | tr '\n' ':')
+      EXPLORE_CLASSPATH=$(ls -1 ${HIVE_HOME}/lib/hive-exec-* ${HIVE_HOME}/lib/*.jar ${TEZ_HOME}/*.jar 2>/dev/null | tr '\n' ':')
       export EXPLORE_CONF_FILES EXPLORE_CLASSPATH
     fi
   fi
```